src/CLITests/OptionsUnitTests.cs

```diff
@@ -36,41 +35,20 @@ private int FailIfCalled(IEnumerable<Error> errors)
             return UnexpectedFailure;
         }
 
-        private int ExpectErrorType(IEnumerable<Error> errors, ErrorType expectedError)
-        {
-            List<Error> errorList = errors.ToList<Error>();
-            Assert.AreEqual(1, errorList.Count);
-            Assert.AreEqual(expectedError, errorList[0].Tag);
-            return ExpectedParseFailure;
-        }
-
         private int ValidateOptions(Options options, string processName = null,
             int processId = 0, string outputDirectory = null, string scanId = null,
-            string verbosity = null)
+            string verbosity = null, bool showThirdPartyNotices = false)
```

Comment:
This lets us leave the mock for ShowThirdPartyNotices unprogrammed in the couple of cases that need it that way